lib/vdsm/storage/mpathconf.py

```diff
@@ -85,9 +86,11 @@ def format_blacklist(wwids):
 
     Returns:
         A formatted blacklist section string.
+        RBD devices were added to the multipath blacklist (BZ#1881832)
     """
     lines = "\n".join('    wwid "{}"'.format(w) for w in sorted(wwids))
     return """blacklist {{
+    devnode "^(rbd)[0-9]*"
```

Comment:
This is create dynamically and is specific for each host, while rbd blacklist is static, always same. I'd consider to move it to [multipath blacklist](https://github.com/oVirt/vdsm/blob/master/lib/vdsm/tool/configurators/multipath.py#L258)

Reply:
I agree with Vojta, this is not the place to put static rules.

Reply:
ACK, shoot in the dark for finding the right place for it :)